File: app/src/main/java/com/dreamland/rxcache/loader/NoMemoryCacheLoader.java
package com.dreamland.rxcache.loader;


import rx.Observable;

/**
 * Created by XMD on 2017/6/15.
 * Only have disk and network
 */
public class NoMemoryCacheLoader extends DefaultCacheLoader {
    public NoMemoryCacheLoader(int maxSize) {

    }

    public NoMemoryCacheLoader() {

    }

    @Override
    public Observable loadFromMemory(Observable observable) {
        return Observable.empty();
    }
}

File: app/src/main/java/com/dreamland/rxcache/loader/NoMemoryCacheLoader.kt
package com.tuyou.tsd.rxcache.loader

import android.content.Context
import com.dreamland.rxcache.loader.DefaultCacheLoader
import com.dreamland.rxcache.rxutils.Tuple3
import rx.Observable

/**
 * Created by XMD on 2017/6/21.
 */
class NoMemoryCacheLoader : DefaultCacheLoader() {
    override fun <T:Any> loadFromMemory(observable: Observable<Tuple3<Context, String, Class<T>>>): Observable<Any?> = Observable.empty()
}